server/controllers/contactController.js

```diff
@@ -0,0 +1,30 @@
+const Contact = require('../models/Contact');
+
+const handleErrors = (err)=>{
+    let errors = {firstName:'',lastName:'',email:'',message:''};
+
+    //validation errors
+    if(err.message.includes('contact validation failed')){
+        Object.values(err.errors).forEach(({properties})=>{
+            errors[properties.path] = properties.message;
+        });
+        return errors;
+    }
+}
+
+const saveContact = async (req,res)=>{
+    const {firstName,lastName,email,message} = req.body;
+    // console.log(firstName,lastName,email,message)
```

Comment:
Remove unecessarry console.log() stements in your final code. They should only be used in your development environment.

Reply:
Removed unnecessary console.log()s statements from the code